README.md

```diff
@@ -98,9 +98,9 @@ The sample app serves as both a reference implementation and a testing tool for
       ```kotlin
       // build.gradle.kts
       dependencies {
-          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:4.2.0-alpha.2")
-          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:4.2.0-alpha.2")
-          implementation("com.github.klaviyo.klaviyo-android-sdk:forms:4.2.0-alpha.2")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:4.2.0")
```

Comment:
alpha.1 and 2 were made for test app builds last week. We'll probably still want to do the LA release as something like `4.2.0-rc.1` but we will make a dedicated branch for that tag.